flexmeasures/api/v3_0/sensors.py

```diff
@@ -576,3 +577,56 @@ def post(self, sensor_data: dict):
         db.session.add(sensor)
         db.session.commit()
         return sensor_schema.dump(sensor), 201
+
+    @route("/<id>", methods=["PATCH"])
+    @use_args(partial_sensor_schema)
+    @use_kwargs({"db_sensor": SensorIdField(data_key="id")}, location="path")
+    @permission_required_for_context("update", ctx_arg_name="db_sensor")
+    @as_json
+    def patch(self, sensor_data: dict, id: int, db_sensor: Sensor):
+        """Update an sensor given its identifier.
+
+        .. :quickref: Sensor; Update a sensor
+
+        This endpoint sets data for an existing sensor.
+        Any subset of sensor fields can be sent.
+
+        The following fields are not allowed to be updated:
+        - id
+        - generic_asset_id
```

Comment:
I believe `entity_address` is also not possible, and it should not.
I believe it is excluded because in the Schema it has `dump_only=True`. Or is using the attribute in `exclude` as you do above the only way?

Please test if that attribute can be sent in.

And we should use only one way. I realize that with the `GenericAssetSchema` we also use both ways. And there we include the `id` field (with `dump_only=True`), which we don't do in the `SensorSchema` as of now. We can send the Sensor Id, so let's add that there, as well.

Reply:
We discussed that there might be good reasons to have both: `dump_only=True` is useful for attributes which should never come in (e.g. ID, generated by the database), and `exclude` is useful for other cases (e.g. PATCH), where the PartialSchema will be used - for attributes which can be POSTed but not PATCHed.